src/ml/neural_net/style_transfer/mps_style_transfer_utils.h

```diff
@@ -0,0 +1,88 @@
+#import <Accelerate/Accelerate.h>
+#import <Foundation/Foundation.h>
+#import <Metal/Metal.h>
+#import <MetalPerformanceShaders/MetalPerformanceShaders.h>
+
+@interface TCMPSConvolutionDescriptor:NSObject
+  @property (nonatomic) NSUInteger kernelWidth;
+  @property (nonatomic) NSUInteger kernelHeight;
+  @property (nonatomic) NSUInteger inputFeatureChannels;
+  @property (nonatomic) NSUInteger outputFeatureChannels;
+  @property (nonatomic) NSUInteger strideWidth;
+  @property (nonatomic) NSUInteger strideHeight;
+  @property (nonatomic) NSUInteger paddingWidth;
+  @property (nonatomic) NSUInteger paddingHeight;
+  @property (nonatomic) NSString *label;
```

Comment:
Nit: In ObjC it's a good habit to use the `copy` attribute for strings. It's a bit subtle, but most of the time this won't actually copy, if the string is immutable.

If it's a mutable string, that's exactly when a copy is desired.